back/src/main/java/sellerlee/back/article/application/ArticleResponse.java

```diff
@@ -35,6 +39,7 @@ public ArticleResponse(Long id, String title, Category category, Long price, Str
 
     public static ArticleResponse of(Article article) {
         return new ArticleResponse(article.getId(),
+                article.getAuthor().getId(),
```

Comment:
```suggestion
                article.getAuthorId(),
```

한 줄에 하나의 점만 존재하도록 메서드를 만들어보는 건 어떤가요?!

Reply:
그렇게하면 Article에 Member의 필드만큼 getter가 생길 것 같아요.
Id에만 그렇게 하는 것에 대핸 어느정도 공감되네요. 이 부분도 내일 얘기해보고 반영할게요 :)